planemo/galaxy/distro_tools.py

```diff
@@ -32,17 +31,30 @@
     "barchart_gnuplot": "plotting/bar_chart.xml",
     "microbial_import1": "data_source/microbial_import.xml",
     "axt_to_concat_fasta": "filters/axt_to_concat_fasta.xml",
-    "__FILTER_FROM_FILE__": "${model_tools_path}/filter_from_file.xml",
     "Interval2Maf_pairwise1": "maf/interval2maf_pairwise.xml",
     "Show beginning1": "filters/headWrapper.xml",
     "axt_to_lav_1": "filters/axt_to_lav.xml",
     "modmine": "data_source/modmine.xml",
-    "__EXPORT_HISTORY__": "${model_tools_path}/imp_exp/exp_history_to_archive.xml",
     "MAF_Thread_For_Species1": "maf/maf_thread_for_species.xml",
     "vcf_to_maf_customtrack1": "maf/vcf_to_maf_customtrack.xml",
     "MAF_To_BED1": "maf/maf_to_bed.xml",
     "__DATA_FETCH__": "${model_tools_path}/data_fetch.xml",
     "__IMPORT_HISTORY__": "${model_tools_path}/imp_exp/imp_history_from_archive.xml",
+    "__EXPORT_HISTORY__": "${model_tools_path}/imp_exp/exp_history_to_archive.xml",
+    "__UNZIP_COLLECTION__": "${model_tools_path}/unzip_collection.xml",
+    "__ZIP_COLLECTION__": "${model_tools_path}/zip_collection.xml",
+    "__FILTER_FAILED_DATASETS__": "${model_tools_path}/filter_failed_collection.xml",
+    "__FILTER_EMPTY_DATASETS__": "${model_tools_path}/filter_empty_collection.xml",
+    "__FLATTEN__": "${model_tools_path}/flatten_collection.xml",
+    "__SORTLIST__": "${model_tools_path}/sort_collection_list.xml",
+    "__MERGE_COLLECTION__": "${model_tools_path}/merge_collection.xml",
+    "__RELABEL_FROM_FILE__": "${model_tools_path}/relabel_from_file.xml",
+    "__FILTER_FROM_FILE__": "${model_tools_path}/filter_from_file.xml",
+    "__TAG_FROM_FILE__": "${model_tools_path}/tag_collection_from_file.xml",
+    "__APPLY_RULES__": "${model_tools_path}/apply_rules.xml",
+    "__BUILD_LIST__": "${model_tools_path}/build_list_1.2.0.xml",
```

Comment:
```suggestion
    "__BUILD_LIST__": ["${model_tools_path}/build_list.xml", "${model_tools_path}/build_list_1.2.0.xml"],
```

I guess it would also be a good time to drop `"filters/catWrapper2.xml"` from line 86, since that has never appeared in a released Galaxy version, as far as I'm aware.